textile/chat-features.textile

```diff
@@ -950,6 +918,10 @@ This section contains error codes that are specific to Chat. If a specific error
 For non-chat-specific codes, the status code for the error should align with the error code. For example, error code @40000@ should hav status code @400@.
 
 <pre>
+
+  // The request was invalid.
+  BadRequest = 40000,
```

Comment:
What was the intention of this error code? It doesn't seem to be referred to in the spec. Also, should CHA-RC1f1 (the only spec point that mentions a code of 40000) be updated to refer to this?

Reply:
Updated CHA-RC1f1 a92c8133